ci-operator/jobs/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-presubmits.yaml

```diff
@@ -0,0 +1,28 @@
+presubmits:
+  openshift/cluster-api-provider-aws:
```

Comment:
Likely want a `--target=[images]` presubmit as well

Reply:
You're right. Added. :+1:

Reply:
I would suggest separate jobs